.github/workflows/gh-pages.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       with:
         submodules: 'recursive'
 
-    - uses: mamba-org/provision-with-micromamba@main
+    - uses: mamba-org/setup-micromamba@v1
```

Comment:
This has already been merged, you need to rebase.

Reply:
I don't think so: https://github.com/BlueBrain/HighFive/blob/master/.github/workflows/gh-pages.yml#L27

Reply:
Only ci.yml has been modified this is gh-pages.yml (for the doc)

Reply:
In that case: Thank you! =)